assets/reader-activation/auth.js

```diff
@@ -293,11 +298,6 @@ window.newspackRAS.push( function ( readerActivation ) {
 					);
 				}
 			} );
-			readerActivation.on( 'reader', () => {
-				if ( readerActivation.getOTPHash() ) {
-					setFormAction( 'otp' );
-				}
-			} );
```

Comment:
It looks like this was causing setFormAction to fire twice during the OTP flow in conjunction with this call here: https://github.com/Automattic/newspack-plugin/blob/c1a68538ac81c242e1103dcaecc707eb1df03005/assets/reader-activation/auth.js#L433

Removing this seems to do the trick to prevent this from being called twice and doesn't appear to break the flow, but I'm not really sure what other contexts this might come into play so would appreciate some additional eyes on this bit!

Reply:
This change was intentional in #2053. However, with the move in RAS-ACC towards detecting whether or not a reader has a password and showing the password or OTP input automatically, I don't think it matters.